### ISSUE TITLE
MD044: HTML comments check is skipped if HTML comment appears as only thing on a line

### ISSUE THREAD

**Issue by eread** [rvben/rumdl#446]
### rumdl version

0.1.25

### Installation method

mise

### Bug description

MD044 https://github.com/rvben/rumdl/blob/ee659eacb863b89d32cb8c64833e9af3c56daa61/docs/md044.md can check for capitalisation of words in `names` configuration.

It can also, by default, check for capitalization of words within HTML comments `<!-- -->`.

However, this check is skipped if the HTML comment is the only thing on the line.

### Steps to reproduce

1. Create a Markdown file with this content:
   ```markdown
   # Heading

   <!-- this is a test example -->
   ```
1. Create a `.rumdl.toml` file with this content:
   ```toml
   [MD044]
   names = ["Test"]
   ```
1. Run `rumdl check` on the Markdown file. No violations are flagged.
1. Modify the Markdown file with the following contents:
   ```markdown
   # Heading

   Test <!-- this is a test example -->
   ```
1. Run `rumdl check` on the Markdown file. The violation inside the HTML comment is flagged.

### Expected behavior

The violation in the sample Markdown file should be found regardless of whether or not it's contained in an HTML comment on its own line.

### Actual behavior

The violation in the sample Markdown file is only found if it's contained in an HTML comment that has other content on the line.

### Configuration

```toml
[MD044]
names = ["Test"]
```

### Operating system

macOS

**Comment by eread** [comment]
@rvben Thanks for fixing this! Seems to work well in version `0.1.26`.

**Comment by rvben** [comment]
Fixed in v0.1.26 @eread. HTML comments are now checked for proper names even when they're the only content on a line.
